acceptance/tests/ensure_puppet_facts_can_use_facter_ng.rb

```diff
@@ -2,6 +2,8 @@
 
   require 'puppet/acceptance/common_utils'
 
+  confine :except, :platform => 'el-5-x86_64'
```

Comment:
should be: `confine :except, :platform => /el-5/`

Reply:
Changed